bpf/tc_common.h

```diff
@@ -0,0 +1,72 @@
+#ifndef TC_COMMON_H
```

Comment:
This code was just extracted from tc_tracer_l7 because we use it now in both places.